src/main/java/study/step1/domain/PositiveNumbers.java

```diff
@@ -0,0 +1,24 @@
+package study.step1.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PositiveNumbers {
+    private final List<PositiveNumber> positiveNumberList;
+
+    public PositiveNumbers(String[] stringNumbers) {
+        positiveNumberList = parsePositiveNumbers(stringNumbers);
+    }
+
+    private List<PositiveNumber> parsePositiveNumbers(String[] stringNumbers) {
+        List<PositiveNumber> numbers = new ArrayList<>();
+        for(String number : stringNumbers) {
+            numbers.add(new PositiveNumber(number));
+        }
+        return numbers;
+    }
+
+    public int sum() {
+        return positiveNumberList.stream().mapToInt(PositiveNumber::getNumber).sum();
```

Comment:
스트림은 행위 단위로 내려쓰는 것이 가독성 향상에 좋습니다 😄

```suggestion
        return positiveNumberList.stream()
                .mapToInt(PositiveNumber::getNumber)
                .sum();
```